facebook_connector.py

```diff
@@ -0,0 +1,110 @@
+"""
+
+Author: Nigel Schuster <nigel schusters at gmail dot com>
+
+This module provides a simple abstraction for accessing the Facebook chat data
+
+"""
+
+from __future__ import print_function
+
+from bs4 import BeautifulSoup
+from fbchat_archive_parser.parser import MessageHtmlParser
+import os
+import pandas as pd
+import re
+import warnings
+
+_messages_file = None
+_fb_id_pattern = re.compile(r"(\d+)@facebook\.com")
+
+
+def initialize(dump_directory="."):
+    """
+        Initializes the access to the Facebook messages file by asserting its location
+
+        Args:
+            dump_directory: path to the directory that contains messages.htm file
+    """
+    global _messages_file
+    fb_message_filename = "messages.htm"
+    _messages_file = os.path.join(dump_directory, fb_message_filename)
+    if not os.path.isfile(_messages_file):
+        print("Please specify the location of the file in the function call")
```

Comment:
Can you recursively walk down from the passed directory?  And also clarify the error message if it's not found?

If you'd rather delay the recursive walk, that's fine, just clarify the error message.  E.g. something like "The location passed did not contain messages.htm, the location of this directory is often '<the unzipped facebook archive>/html'